src/gui/qgsscalewidget.h

```diff
@@ -32,6 +32,8 @@ class GUI_EXPORT QgsScaleWidget : public QWidget
 {
     Q_OBJECT
     Q_PROPERTY( bool showCurrentScaleButton READ showCurrentScaleButton WRITE setShowCurrentScaleButton )
+    Q_PROPERTY( bool scale READ scale WRITE setScale )
```

Comment:
@m-kuhn, my understanding is that NOTIFY is needed only if MEMBER is used. This is not the case, here.